docs/tables.md

```diff
@@ -35,25 +36,32 @@ Tables have been simplified to mirror a more standard implementation. We've also
   --table-head: #5b1c9f;
   --table-head-text: white;
   --table-stripe: #f0eaf7;
-  --table-edges: rgba(34, 5, 64, .5);
+  --table-edges: rgba(34, 5, 64, 0.5);
   --table-row: white;
 }
 ```
+
 ```scss CSS Selectors
 /* Table
  */
-.markdown-body .rdmd-table table {}
+.markdown-body .rdmd-table table {
+}
 
 /* Rows
  */
-.markdown-body .rdmd-table tr {}
-.markdown-body .rdmd-table thead tr {} /* header row's background */
-.markdown-body .rdmd-table tr:nth-child(2n) {} /* striped rows' background */
+.markdown-body .rdmd-table tr {
+}
+.markdown-body .rdmd-table thead tr {
+} /* header row's background */
+.markdown-body .rdmd-table tr:nth-child(2n) {
+} /* striped rows' background */
 
 /* Cells
  */
-.markdown-body .rdmd-table th {}
-.markdown-body .rdmd-table td {}
+.markdown-body .rdmd-table th {
+}
+.markdown-body .rdmd-table td {
+}
 ```
 
 <style>
```

Comment:
doesn't look like `acorn` is a big fan of this `<style>` tag:

<img width="590" alt="image" src="https://github.com/readmeio/markdown/assets/20567510/1f431c2c-0808-4b42-b954-1be511a92526">

but haaaay nice error messaging!

Reply:
It's not so much the script, but css. I imagine that MDX looked at trying to detect and parse css, and just table flipped